### PR TITLE
Refactoring

### LINKED ISSUE #777: Avoid redundant navigation to current location TipRecord, TipComment

### LINKED ISSUE #779: Make AuthorAndDate component looks the same in all cases
@onvisions @mradkov @davidyuk this is a continuation of the [previous discussion](https://github.com/aeternity/superhero-ui/pull/771#discussion_r477321797)

image
https://develop.aggregator.z52da5wt.xyz/tip/1818

### LINKED ISSUE #782: TipRecord: sendReport doesn't work if user is not logged in
https://github.com/aeternity/superhero-ui/blame/develop/src/utils/backendAuthMixin.js#L34
https://github.com/aeternity/superhero-ui/blob/develop/src/components/tipRecords/TipRecord.vue#L21

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #782,
fixes #779,
fixes #777,
Fix the align of the loading
|old|new|
|---|----|
|image|image|

### CODE REVIEW

File: src/views/Maintenance.vue [hunk 1]
```diff
@@ -10,7 +10,6 @@
 import { EventBus } from '../utils/eventBus';
 
 export default {
-  name: 'Maintenance',
```
Comment (mradkov) [inline src/views/Maintenance.vue:13]: Is there any specific reason to remove implicit names for components and views?

Comment (CedrikNikita) [reply src/views/Maintenance.vue:13]: I suggest deleting them since we never refer to them by name. Plus, there is another opportunity to get a typo.

**Review by mradkov — CHANGES_REQUESTED** [review]
There is a visual issue with the Tip button displayed in the UserProfile:
Screenshot 2020-09-04 at 14 21 42

**Comment by CedrikNikita** [comment]
> There is a visual issue with the Tip button displayed in the UserProfile:
> Screenshot 2020-09-04 at 14 21 42

It's on develop, it's not related

**Review by mradkov — APPROVED** [review]

**Pull request merged**